mysql-test/suite/galera_3nodes/t/galera_gtid_2_cluster.test

```diff
@@ -282,6 +290,7 @@ select @@gtid_binlog_state;
 --source include/save_master_gtid.inc
 --connection node_1
 --source include/sync_with_master_gtid.inc
+select * from t1 order by 1, 2, 3;
 
 
 --echo cluster 1 node 1
```

Comment:
It would be nice that you would verify on every node
(1) t1 has identical contents on every node
(2) current GTID is consistent